### DISCUSSION TITLE
1.19.1 (17 July 2023)

### DISCUSSION THREAD

**Discussion by johnwalley** [johnwalley/allotment#663] [Announcements]
### Bug Fixes

* forward ref to underlying Pane dom element ([#632](https://github.com/johnwalley/allotment/issues/632)) ([574c1ef](https://github.com/johnwalley/allotment/commit/574c1efcf7c3cc26656a074450a29f16168ac790))

<hr /><em>This discussion was created from the release <a href='https://github.com/johnwalley/allotment/releases/tag/v1.19.1'>1.19.1 (17 July 2023)</a>.</em>

**Comment by IgnacioLura** [comment]
Hello community!

When I try to deploy my pipeline, the "npm install" command fails for this specific package. Has anyone else experienced a similar issue?

Thanks!

edit: for this new version only!

**Reply by johnwalley** [reply]
Could you try v1.19.2 (I'm not happy with the testing I did for v1.19.1 so I've reverted a change)? If it is still not working could you let me know which versions of node and npm you are using? Thanks!

**Reply by IgnacioLura** [reply]
Sure! I let you know next time I have to do a deploy, thanks
